MIGRATIONS.md

```diff
@@ -3,6 +3,13 @@
 This file documents changes in the data model. Please explain any changes to the
 data model as well as any custom migrations.
 
+## WordPress 154
+
+@momozw 2024-05-07
+
+- `BasePost`: 
```

Comment:
It should be added to `AbstractPost` because `BasePost` is also a base class for posts in the reader.

Reply:
Fixed [b03e68d](https://github.com/wordpress-mobile/WordPress-iOS/pull/23173/commits/b03e68db7ab592db017653fbb1ab1833bb08811d)